src/main/java/seedu/address/model/person/Designation.java

```diff
@@ -0,0 +1,51 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Person's designation in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidDesignation(String)}
+ */
+public class Designation {
+
+    public static final String MESSAGE_DESIGNATION_CONSTRAINTS =
+            "Designation can only be either 'manager' or 'employee'";
+    public static final String DESIGNATION_VALIDATION_REGEX = "^[A-Za-z]+$";
+    public final String value;
+
+    /**
+     * Constructs a {@code Designation}.
+     *
+     * @param designation A valid designation.
+     */
+    public Designation(String designation) {
+        requireNonNull(designation);
+        checkArgument(isValidDesignation(designation), MESSAGE_DESIGNATION_CONSTRAINTS);
+        value = designation;
+    }
+
+    /**
+     * Returns true if a given string is a valid phone number.
+     */
+    public static boolean isValidDesignation(String test) {
+        return test.matches(DESIGNATION_VALIDATION_REGEX);
```

Comment:
if it can only be 'manager' or 'employee', is the designation valid regex necessary?